src/pybind/mgr/prometheus/module.py

```diff
@@ -86,8 +85,8 @@ def __init__(self, mtype, name, desc, labels=None):
         self.mtype = mtype
         self.name = name
         self.desc = desc
-        self.labelnames = labels    # tuple if present
-        self.value = {}             # indexed by label values
+        self.labelnames = labels  # tuple if present
```

Comment:
what is this change for?

Reply:
ok

Reply:
"ok" as in "I'll remove the change", or?...

Reply:
I think the self.value init is required

Reply:
@dmick Yes, I had remove the change, `self.value` is initialize at [here](https://github.com/iliul/ceph/blob/8b868fec151d30937cf37cb5042dc7be2dd25773/src/pybind/mgr/prometheus/module.py#L89)